README.md

```diff
@@ -65,6 +44,58 @@ And when you're done, spin down mysql (it is also fine to leave it running for y
 ./docker/run-mysql.sh stop
 ```
 
+## Running Locally
+
+### Requirements:
+
+* [Docker Desktop](https://www.docker.com/products/docker-desktop) (4GB+, 8GB recommended)
+* Broad internal internet connection (or VPN, non-split recommended)
+* Render the local configuration files. From the root of the [firecloud-develop](https://github.com/broadinstitute/firecloud-develop) repo, run:
+```sh
+sh run-context/local/scripts/firecloud-setup.sh
+```
+Note: this script will offer to set up configuration for several other services as well. You can skip those if you only want to set up configuration for Rawls. If this is your first time running Rawls or rendering congiguration files, you will want to run through the "Setup vault" step.
```

Comment:
Nit: congiguration -> configuration in the second sentence :)

Reply:
how embarrassing. just fixed!